### PR TITLE
Auth work

### PR DESCRIPTION
**Pull request by Aram-Anderson**
#
?
Description of Changes Made: 
Added migration to users for plat admin column
Anticipated Code Effects: 
(Would this merge affect any other piece of our code?)
Probably will be some fire somewhere
Reviewer: 
@djtrujillo @snayrouz @sidewinder2020

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def require_admin
   end
 
   def authorize!
-    permission = PermissionService.new(current_user, params[:store_slug], params[:controller], params[:action])
+    permission = PermissionService.new(current_user, params[:controller], params[:action])
```
Comment (snayrouz) [inline app/controllers/application_controller.rb:34]: wow great job aram!

**Review by djtrujillo — APPROVED** [review]

File: app/views/store/manager/items/index.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      <th>Price</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @items.each do |item| %>
+    <tr class="item">
+      <td><%= item.title %></td>
+      <td><%= item.price %></td>
+      <td><%= truncate(item.description) %></td>
```
Comment (snayrouz) [inline app/views/store/manager/items/index.erb:16]: love it!

**Review by snayrouz — APPROVED** [review]
great work! someone else can merge

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (sidewinder2020) [reply app/controllers/application_controller.rb:34]: awesome - I'm happier with that slug outta there

File: db/migrate/20171023182538_add_plat_admin_to_user.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddPlatAdminToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :platform_admin, :boolean, default: false
```
Comment (sidewinder2020) [inline db/migrate/20171023182538_add_plat_admin_to_user.rb:3]: toggle toggle

**Review by sidewinder2020 — COMMENTED** [review]
Love it, let's do this

**Pull request merged**